packages/clima_ui/lib/widgets/dialogs/api_key/api_key_reset_dialog.dart

```diff
@@ -24,7 +28,15 @@ class ApiKeyResetDialog extends StatelessWidget {
             ),
           ),
           TextButton(
-            onPressed: () {
+            onPressed: () async {
+              await ref
+                  .read(apiKeyStateNotifierProvider.notifier)
+                  .setApiKey(const ApiKeyModel.default_());
+              showSnackBar(
+                context,
+                text: 'API key reset successfully.',
```

Comment:
I think this text is okay, but if you have any suggestions feel free to give them.